### PR TITLE
Feature: subcommands flags inherence

### PR DESCRIPTION
**Pull request by thetechpanda**
Implements flags inheritance from parent command.

### CODE REVIEW

File: command_test.go [hunk 1]
```diff
@@ -131,6 +131,15 @@ func TestCommand(t *testing.T) {
 
 	})
 
+	t.Run("Run NewSubCommandInheritFlags()", func(t *testing.T) {
+		c.NewSubCommandInheritFlags("name", "description")
+	})
+
+	t.Run("Run inheritFlags()", func(t *testing.T) {
+		s := c.NewSubCommandInheritFlags("name", "description")
+		s.inheritFlags(c.flags)
```
Comment (leaanthony) [inline command_test.go:140]: We should add a test that the flag is actually on `s` and `t.Fail()` if it is not 👍

**Comment by leaanthony** [comment]
Looking good my friend 👍 Just a little comment on the test but LGTM 🚀

File: command_test.go (hunk 1 above)
Comment (thetechpanda) [reply command_test.go:148]: I've added tests for both `NewSubCommandInheritFlags()` and `inheritFlags()`

**Comment by leaanthony** [comment]
Awesome! Thanks man!

**Pull request merged**